Source/OIDTokenUtilities.m

```diff
@@ -49,7 +49,7 @@ + (NSData *)sha265:(NSString *)inputString {
 }
 
 + (NSString *)truncate:(NSString *)inputString {
-    return [inputString length] >= 6 ? [[inputString substringToIndex:6] stringByAppendingString:@"..."] : @"...";
+    return [inputString length] >= 9 ? [[inputString substringToIndex:6] stringByAppendingString:@"..."] : @"[redacted]";
```

Comment:
Sorry for the nits, but a few minor code formatting issues...

- We always use dot notation for properties, so `inputString.length` instead of `[inputString length]`.
- We only use 2 spaces for indentation, not 4 (please fix here and in the tests).
- We have a max line length of 100 columns. Line continuations are indented +4, so this should be:

```
+ (NSString *)truncate:(NSString *)inputString {
  return [inputString length] >= 9
      ? [[inputString substringToIndex:6] stringByAppendingString:@"..."] : @"[redacted]";
}

```